src/App.tsx

```diff
@@ -15,71 +16,87 @@ export const goods = [
   'Garlic',
 ];
 
-export const App: React.FC = () => (
-  <main className="section container">
-    <h1 className="title">No goods selected</h1>
+export class App extends React.Component {
+  state = {
+    selectedWord: 'Jam',
+  };
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  render(): React.ReactNode {
+    return (
+      <main className="section container">
+        {
+          !this.state.selectedWord
+            ? (
+              <h1 className="title">No goods selected</h1>)
+            : (
+              <h1 className="title is-flex is-align-items-center">
+                {`${this.state.selectedWord} `}
+                is selected
 
-      {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
+                {/* eslint-disable-next-line jsx-a11y/control-has-associated-label  */}
+                <button
+                  data-cy="ClearButton"
+                  type="button"
+                  className="delete ml-3"
+                  onClick={() => (
+                    this.setState({ selectedWord: '' })
+                  )}
+                />
+              </h1>
+            )
+        }
 
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
-            >
-              +
-            </button>
-          </td>
+        <table className="table">
+          <tbody>
+            {
+              goods.map(good => (
+                <tr
+                  data-cy="Good"
+                  key={good}
+                  className={classNames(
+                    {
+                      'has-background-success-light':
+                      this.state.selectedWord === good,
+                    },
+                  )}
+                >
+                  <td>
+                    <button
+                      data-cy={
+                        this.state.selectedWord === good
```

Comment:
you've used this check 5 times, let's move it to constant
also would be nice to create constants for buttons text, callback, data-cy instead of using ternary operators everywhere, but this point is less critical